package-lock.json

```diff
@@ -6672,12 +6672,14 @@
         "balanced-match": {
```

Comment:
Let's remove `package-lock` changes from this PR since we don't modify `package.json` neither: changes seem to be just differences in NPMs of macos/linux.

Reply:
Done. Thank your for the suggestion, I didn't know that one.